src/http/server/handlers/mod.rs

```diff
@@ -58,6 +73,31 @@ impl<B: Send> FromRequest<B> for QueryConfigFromReq {
     }
 }
 
+struct ByteArrStreamFromReq(ByteArrStream);
+
+#[cfg(feature = "enable-serde")]
+#[async_trait]
+impl<B: HttpBody<Data = Bytes, Error = hyper::Error> + Send + 'static> FromRequest<B>
+    for ByteArrStreamFromReq
+{
+    type Rejection = Error;
+
+    async fn from_request(req: &mut RequestParts<B>) -> Result<Self, Self::Rejection> {
+        #[derive(serde::Deserialize)]
+        struct FieldTypeParam {
+            field_type: FieldType,
+        }
+
+        let Query(FieldTypeParam { field_type }) = req.extract().await?;
+        let body: BodyStream = req.extract().await?;
+        // TODO: multiply the field_type by 22; this is a hacky way of making sure IPA can run,
```

Comment:
can you follow up and make a small change here?

this todo is very important to address, so I am thinking that we should also print a warning each time this code is executed

Reply:
following up to say this was fixed